content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/embeddable/youtube/v1/.content.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This might break existing applications based on `.../embed/embeddable/youtube`.
I would not introduce a v1 version as this might confuse developers.
I would:
- keep the existing `.../embed/embeddable/youtube` embeddable
- introduce a new v2 youtube embeddable at `.../embed/embeddable/youtube/v2/youtube`.
Please note that the new embedabble should follow the component versioning pattern and should be at `.../embed/embeddable/youtube/v2/youtube` not at `.../embed/embeddable/youtube/v2` as currently implemented in this PR